sdk/tests/conformance2/sync/sync-webgl-specific.html

```diff
@@ -79,7 +79,7 @@
   var iter = numRetries;
   while (--iter > 0) {
     syncWithGLServer();
-    if (gl.getSyncParameter(sync, gl.SYNC_STATUS) == gl.SYNC_SIGNALED) {
+    if (gl.getSyncParameter(sync, gl.SYNC_STATUS) == gl.SIGNALED) {
```

Comment:
Should this ALWAYS return UNSIGNALED? `!== gl.UNSIGNALED` would be more strict, if it would work.